ch_tools/common/logging.py

```diff
@@ -0,0 +1,189 @@
+    loguru_handlers = []
+
+    for name, value in config_loguru["handlers"].get(module, {}).items():
+        format_ = config_loguru["formatters"][value["format"]]
+        handler = {
+            "sink": value["sink"],
+            "format": format_,
+            "enqueue": True,
+            "diagnose": False,
+        }
+        if "level" in value:
+            handler["level"] = value["level"]
+
+        if "filter" in value:
+            if value["filter"]:
+                handler["filter"] = make_filter(value["filter"])
+                # https://loguru.readthedocs.io/en/stable/api/logger.html#message
+                # One can also pass a dict mapping module names to minimum required level. In such case, each log record will search for it’s closest parent in the dict
+                # and use the associated level as the filter.
+                # In order to set a default level, the "" module name should be used as it is the parent of all modules (it does not suppress global level threshold, though).
+                handler["filter"][""] = False
+        else:
+            handler["filter"] = make_filter(name)
+        loguru_handlers.append(handler)
+
+    logger.configure(handlers=loguru_handlers, activation=[("", True)])
+
+    # Configure logging.
+    logging.basicConfig(handlers=[InterceptHandler()], level=0)
+
+
+def critical(msg, *args, **kwargs):
+    """
+    Log a message with severity 'CRITICAL'.
+    """
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).critical(
+        msg, *args, **kwargs
+    )
+
+
+def error(msg, *args, **kwargs):
+    """
+    Log a message with severity 'ERROR'.
+    """
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).error(msg, *args, **kwargs)
+
+
+def exception(msg, *args, **kwargs):
+    """
+    Log a message with severity 'ERROR' with exception information.
+    """
+
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).debug(msg, *args, **kwargs)
+
+
+def warning(msg, *args, **kwargs):
+    """
+    Log a message with severity 'WARNING'.
+    """
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).warning(
+        msg, *args, **kwargs
+    )
+
+
+def info(msg, *args, **kwargs):
+    """
+    Log a message with severity 'INFO'.
+    """
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).info(msg, *args, **kwargs)
+
+
+def debug(msg, *args, **kwargs):
+    """
+    Log a message with severity 'DEBUG'.
+    """
+    with_exception = kwargs.get("exc_info", False)
+    getLogger(this["module"]).opt(exception=with_exception).debug(msg, *args, **kwargs)
+
+
+def log_status(status, msg):
+    """
+    Log a message with by status code.
+    """
+    if status == 0:
```

Comment:
Suggest to use constants OK, WARNING and ERROR from ch_tools/common/result.py instead of magic constants.

See also https://github.com/yandex/ch-tools/pull/143 This PR replaces 0, 1, 2 with constants in almost all places in the repo.